net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/stats/views/AbstractStatsView.java

```diff
@@ -296,7 +298,7 @@ protected final IDialogSettings getDialogSettings() {
    */
   private void focusSelectionChanged(IWorkbenchPart part, ISelection selection) {
 
-    List resources = new ArrayList();
+    List<IResource> resources = new ArrayList<IResource>();
```

Comment:
new ArrayList<>(); - but never mind... ;-)

Reply:
I'm still not sure why Eclipse does this sometimes when adding generic arguments. Must be my fault. Besides that, I still want to run some cleanup actions on the current code, where it should be more easy to remove this again, so I would leave it to that.